src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,269 @@
+                content: str = file.read()
+            return content
+        return ""
+
+    def _loadFullStdout(self):
+        """
+        Returns the stdout output of the executable.
+        """
+        filePath: str = self._getStdoutFilePath()
+        return self.__loadFileContent(filePath)
+
+    def _loadFullStderr(self):
```

Ruff

Missing return type annotation for private function `_loadFullStderr`

(ANN202)